packages/fluentui/docs/src/examples/components/Carousel/Variations/CarouselThumbnailsExample.shorthand.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { Carousel, Image } from '@fluentui/react-northstar';
+
+const imageAltTags = {
+  ade: 'Portrait of Ade',
+  elliot: 'Portrait of Elliot',
+  kristy: 'Portrait of Kristy',
+  nan: 'Portrait of Nan',
+};
+const carouselItems = [
+  {
+    key: 'ade',
+    id: 'ade',
+    content: <Image src="public/images/avatar/large/ade.jpg" fluid alt={imageAltTags.ade} />,
+    thumbnail: <Image src="public/images/avatar/small/ade.jpg" fluid alt={imageAltTags.ade} />,
+  },
+  {
+    key: 'elliot',
+    id: 'elliot',
+    content: <Image src="public/images/avatar/large/elliot.jpg" fluid alt={imageAltTags.elliot} />,
+    thumbnail: <Image src="public/images/avatar/small/elliot.jpg" fluid alt={imageAltTags.elliot} />,
+  },
+  {
+    key: 'kristy',
+    id: 'kristy',
+    content: <Image src="public/images/avatar/large/ade.jpg" fluid alt={imageAltTags.ade} />,
+    thumbnail: <Image src="public/images/avatar/small/ade.jpg" fluid alt={imageAltTags.ade} />,
+  },
+  {
+    key: 'nan',
+    id: 'nan',
+    content: <Image src="public/images/avatar/large/nan.jpg" fluid alt={imageAltTags.nan} />,
+    thumbnail: <Image src="public/images/avatar/small/nan.jpg" fluid alt={imageAltTags.nan} />,
+  },
+  {
+    key: 'elliot1',
+    id: 'elliot1',
+    content: <Image src="public/images/avatar/large/elliot.jpg" fluid alt={imageAltTags.elliot} />,
+    thumbnail: <Image src="public/images/avatar/small/elliot.jpg" fluid alt={imageAltTags.elliot} />,
+  },
+  {
+    key: 'ade1',
+    id: 'ade1',
+    content: <Image src="public/images/avatar/large/ade.jpg" fluid alt={imageAltTags.ade} />,
+    thumbnail: <Image src="public/images/avatar/small/ade.jpg" fluid alt={imageAltTags.ade} />,
+  },
+];
+
+const CarouselExample = () => (
+  <Carousel
+    ariaRoleDescription="carousel"
+    ariaLabel="Portrait collection"
+    thumbnails
+    navigation={{
+      'aria-label': 'people portraits',
+      items: carouselItems.map((item, index) => ({
+        key: index,
+        'aria-controls': item.id,
+        'aria-label': imageAltTags[item.id],
+        content: item.thumbnail,
```

Comment:
since here it's the same you can just probably reference the content. Or create another example with a complex carousel structure and have only the image as thumbnail. Or do that in this example.

Leaving it like this is probably the least desirable option

Reply:
The `content` and the `thumbnail` has different sizes of image since we a not going to crop or do something like it for the images. Not sure if I understood what you mean by "Or create another example with a complex carousel structure and have only the image as thumbnail."

Reply:
Oh right, I did not see the large/small difference.